addon/webextension/background/startBackground.js

```diff
@@ -31,6 +31,18 @@ this.startBackground = (function() {
   // Maximum milliseconds to wait before checking for migration possibility
   const CHECK_MIGRATION_DELAY = 2000;
 
+  browser.tabs.onUpdated.addListener((id, changeInfo, tab) => {
+    browser.pageAction.show(tab.id);
+  });
```

Comment:
Do we need this? I'm pretty sure there's a substantial performance impact to using `browser.tabs.unUpdated`